streamline_training.py

```diff
@@ -111,20 +163,32 @@ def __preprocess(args):
                      " --output_tfrecord=" + output_dir + "/tune.tf_record" +
                      " --label_map_file=" + output_file +
                      " --vocab_file=" + bert_dir + "/vocab.txt" +
-                     " --output_arbitrary_targets_for_infeasible_examples=true").split(),
+                     " --output_arbitrary_targets_for_infeasible_examples=true" +
+                     " --embedding_type=" + args.embedding_type).split(),
                     cwd=lasertagger_dir)
 
     subprocess.call(("python preprocess_main.py --input_file=" + training_file +
                      " --input_format=wikisplit" +
                      " --output_tfrecord=" + output_dir + "/train.tf_record" +
                      " --label_map_file=" + output_file +
                      " --vocab_file=" + bert_dir + "/vocab.txt" +
-                     " --output_arbitrary_targets_for_infeasible_examples=false").split(),
+                     " --output_arbitrary_targets_for_infeasible_examples=false" +
+                     " --embedding_type=" + args.embedding_type).split(),
                     cwd=lasertagger_dir)
 
 
 def __training(args):
     """ Train the LaserTagger model."""
+    print("------ Generating config file ------")
+    
+    if args.embedding_type == "POS":
+        bert_type = "POS"
+    else:
+        bert_type = "Base"
+    
+    write_lasertagger_config(output_dir, bert_type, args.t2t, args.number_layer, args.hidden_size, args.num_attention_head, args.filter_size, args.full_attention)
+    config_file_path = "{}/bert_config.json".format(output_dir)
+    
     print("------ Start training ------")
     f = open(output_dir + "/train.tf_record.num_examples.txt", "r")
```

Comment:
use with statement to narrow the scope of the variables

with open(output_dir + "/train.tf_record.num_examples.txt", "r") as f:
  num_train_examples = f.read()

Reply:
Done.